content/en/about-filecoin/what-is-filecoin/index.md

```diff
@@ -37,6 +37,8 @@ Storage providers are people and organizations that run storage providers, earni
 
 Once a storage provider implements the Filecoin protocol, they gain access to the entire market of Filecoin users. Providers don’t need to design their own storage API or advertise their offering themselves as this is handled by Filecoin’s protocol and network. Filecoin enables a thriving, decentralized ecosystem of independent storage providers by removing barriers to entry.
 
+See [How providing storage works](https://docs.filecoin.io/storage-provider/how-providing-works/)
```

Comment:
```suggestion
See [How providing storage works](https://docs.filecoin.io/storage-provider/how-providing-works/)
```

Please use `relref` links. See the Hugo docs for how to format them: https://gohugo.io/functions/relref/#readout